content/providers/migration.md

```diff
@@ -19,7 +19,165 @@ Documentation coming soon.
 For manual migration to the family Official Providers refer to the
 [migration guide]({{<ref "/knowledge-base/migrate-to-provider-families">}}).
 
-Documentation for the automated tooling coming soon.
+`family-migrator` is a tooling for the automated migration process. The tool
+supports interactive and non-interactive migration to the family Official
+Providers. The instructions on how to use tooling are below.
+
+The `family-migrator` tool has two sub-commands, `generate` and `execute`.
+
+<!-- vale alex.ProfanityUnlikely = NO -->
+<!-- ignore "executed" -->
+- `generate`: For generating the migration plan. After the tool creates the
+migration plan, asks if the user wants to continue to perform the created plan.
+
+- `execute`: For only performing the generated plan. To run this command,
+you must have a generated plan. Users can use this option for a generated but
+un-executed plan.
+<!-- vale alex.ProfanityUnlikely = YES -->
```

Comment:
The migration tool generates a plan based on the resources available in the file system and cluster. For example, a migration plan can be generated without any MR in the cluster or a configuration package in the File System. Therefore, I think it might be confusing to add something here, as the situation may differ in each customer environment.

Reply:
We need to look to document the scenario we support up front.